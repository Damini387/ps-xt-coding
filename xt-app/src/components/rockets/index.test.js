import React from 'react';
import { shallow } from 'enzyme';
import Rockets from './index';
import Rocket from './rocket';

const testRocket = [{"mission_name": "test", "flight_number": "1", "mission_id": "TEST", "launch_year": "2050", "launch_success": true, "links" : {"mission_patch": "image.jpg"}}]

let empty_wrapper = shallow(
    <Rockets />);

    
let wrapper = shallow(
    <Rockets rocketData={testRocket}/>);

describe('Rockets', () => {

	it('should render the multiple Rockets Component correctly', () => {
		expect(empty_wrapper).toMatchSnapshot();
	});

	it('should render Filters and Rockets component correctly', () => {
        expect(wrapper.find(Rocket)).toHaveLength(1);
	});

});