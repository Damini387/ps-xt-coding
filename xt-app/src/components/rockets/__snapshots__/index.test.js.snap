// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets should render the multiple Rockets Component correctly 1`] = `
<div
  className="rockets empty"
>
  <div
    className="loader"
  >
    loading...
  </div>
</div>
`;
